tests/governance.py

```diff
@@ -174,13 +175,13 @@ def test_node_ids(network, args):
 
 @reqs.description("Checking service principals proposals")
 def test_service_principals(network, args):
-    primary, _ = network.find_nodes()
+    node = args.choose_node(network)
```

Comment:
Why is this `args.choose_node` rather than `network.find_any_node()`?

Reply:
This is 50/50 primary/backup, and will break if the network does not in fact have backups, so it's a bit better in making sure both cases are equally tested.

Reply:
Moved as discussed.